docs/running-on-mesos.md

```diff
@@ -479,6 +479,35 @@ See the [configuration page](configuration.html) for information on Spark config
 </tr>
 
 <tr>
+  <td><code>spark.mesos.driver.secret.envkey</code></td>
+  <td><code>(none)</code></td>
+  <td>
+    If set, the contents of the secret referenced by
+    spark.mesos.driver.secret.name will be written to the provided
+    environment variable in the driver's process.
+  </td>
+  </tr>
+  <tr>
+<td><code>spark.mesos.driver.secret.filename</code></td>
+  <td><code>(none)</code></td>
+  <td>
+    If set, the contents of the secret referenced by
+    spark.mesos.driver.secret.name will be written to the provided
+    file.  Relative paths are relative to the container's work
+    directory.  Absolute paths must already exist.  Consult the Mesos Secret
+    protobuf for more information.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.mesos.driver.secret.name</code></td>
```

Comment:
@susanxhuynh the type would only be set on the secret `name` for example
```
...
--conf spark.mesos.driver.secret.name=/mysecret:REFERENCE \
--conf spark.mesos.driver.secret.filename=asecret
```

Also, while we're on the topic, what about multiple secrets? Is comma-separated entries the typical way? E.g:
```
--conf spark.mesos.driver.secret.name=/mysecret:REFERENCE,/super/secret:REFERENCE \
--conf spark.mesos.driver.secret.filename=asecret,topsecret
```